openkfw/TruBudget
# 2015: Refactior styles for DocumentUpload modal

### Checklist




### Description
- changed styles for DocumentUpload modal, so it looks and feels better for users
<img width="530" alt="Screenshot 2024-10-08 at 08 47 07" src="https://github.com/user-attachments/assets/7a7e47ea-7529-4f0f-aacb-081e2637dc4b">
<img width="602" alt="Screenshot 2024-10-08 at 08 47 51" src="https://github.com/user-attachments/assets/ef4910d1-3837-4d0a-91eb-b86d78564598">

### How to test

1. <E.g. login as user XYZ>
2. Go to Subprojects and create/edit worklfow action
3. Go to step 2 in modal and see that styles are changed for Document upload as seen in screenshots

Closes #2015
